src/crypto-api.ts

```diff
@@ -591,6 +591,20 @@ export interface ImportRoomKeyProgressData {
     total: number;
 }
 
+/**
+ * Where a room key came from
+ */
+export enum RoomKeySource {
```

Comment:
Feels a bit strange to define all these variant that are not used.
Shouldn't the react sdk use `Export` when importing from file? I don't see the Forward/Direct used also